src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,53 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
+
+    Animal.alive.push(this);
+  }
+
+  decreaseHealth(amount = 50) {
```

Comment:
decreaseHealth method should be private, consider renaming it to _decreaseHealth.